Documentation/azure-devops-as-ci/azure-devops-as-ci.md

```diff
@@ -0,0 +1,56 @@
+Testing with Azure DevOps
+===
+
+We enabled Azure DevOps for pull request validation. You can now see the new "badge" **coreclr-ci** that was added on a pull request page.
+
+We disabled default triggering for the following innerloop jobs in Jenkins:
+
+* Ubuntu arm Cross Checked Innerloop Build and Test
+* Ubuntu arm Cross Checked no_tiered_compilation_innerloop Build and Test
+* Ubuntu16.04 arm64 Cross Checked Innerloop Build and Test 
+* Ubuntu16.04 arm64 Cross Checked no_tiered_compilation_innerloop Build and Test
+* Windows_NT arm Cross Debug Innerloop Build
+* Windows_NT arm Cross Checked Innerloop Build and Test
+* Windows_NT arm64 Cross Debug Innerloop Build
+* Windows_NT arm64 Cross Checked Innerloop Build and Test
+
+since they are using limited hardware.
+
+The results for those jobs can be found in the new CI system. 
+Other Jenkins jobs are running as usual and we will continue monitoring them to validate consistency between Jenkins and AzDev test jobs.
+
+# How to find the test results in Azure DevOps?
+
+If you click on coreclr-ci Details link 
+![coreclr-ci-badge](coreclr-ci-badge.png)
```

Comment:
Do we really need MB worth of pictures in the repo - these pictures will be obsolete in a few weeks or months as the UI develops.

Reply:
I agree we don't need all those pictures and the only reason why I opened this is to have a way for contributor to find their test result in the new CI system while we are testing and adjusting it.